locked-asset/proxy_dex/src/attr_ex_helper.rs

```diff
@@ -0,0 +1,76 @@
+use common_structs::LockedAssetTokenAttributes;
```

Comment:
Make the module public in `factory` and import it from there, let's not copy-paste code unless absolutely necessary.

Reply:
Done.